README.md

```diff
@@ -44,54 +62,126 @@ This goes both ways: when explaining something make sure to present enough conte
+
+Additionally, we record best practices as it relates to groom and sprint planning in our [Sprint learnings document](https://docs.google.com/document/d/1SiRF0R73Tg0IBHHCzX5PWAfyXF_Q9TUogxxOyxR-pAM/edit#heading=h.r7t77g3jnbng).
+
+### Ticket Management
+
+All developers are expected to keep their ticket statuses up to date at all times. Just as our communication section details, no one should have to waste time figuring out the state of work. PRs should be linked to the ticket, and questions and comments should be added to the ticket as well. 
+
+Additionally when a ticket is ready for Quality Assurance (QA), notes on how to QA/verify your work must be added as a comment to the ticket, so the person QA-ing the work doesn't have to ask or guess how to verify your work.
 
-All of our code is open source and on GitHub. We organize stories and plan sprints with Pivotal Tracker, and maintain our roadmaps \(currently\) in Google Spreadsheets . Individual coding tasks are tracked as Pivotal cards. We use pull requests to review and merge changes. Every PR is peer-reviewed. Please see here for an overview of our [Teams, Services, and Environments](http://docs.dosomething.org/engineering/overview). We communicate on [Slack](slack.md).
+Sharing the responsibility for our work extends to ticket management as well. If you see that there's an issue that needs to be captured in a ticket, make the ticket. If you *truly* don't have time to do this, ask your team for help. 
 
-### Code review and pull requests
+#### Pivotal Tracker
+The following outlines the stages of a ticket in Pivotal Tracker. The buttons on a ticket are _Start_, _Finish_, _Deliver_, _Accept_ OR _Reject_, and they will transition a ticket within the different stages.
 
-We work in branches, generally off of dev \(or master, depending on the project\). We commit often, to capture our thinking. We package up our work as pull requests, and try to keep PRs reasonably small. \(This also makes it more likely someone will review your PR!\)
+##### ⭐️ Feature OR 🐞 Bug Ticket
+1. **Planned**: Ready for current iteration (also estimated if a Feature ticket).
+2. **Unstarted**: No work has begun to address the ticket.
+3. **Started**: Work has begun to address the ticket. In order to start the ticket, it must be estimated first.
+4. **Finished**: Work to address ticket is completed. A pull request has been submitted and is awaiting review.
+5. **Delivered**: The pull request has been merged and deployed to a QA environment. The developer has verified their work on this environment and added QA / testing notes to the ticket. At this stage, there will be review by Product, Design, and/or Data. Just as we expect developers to keep their ticket statuses up to date, we expect the same from Reviewers (Unstarted, In Review, Pass, or Revise).
```

Comment:
Worth calling out that the Dev should @ mention any reviewers to make sure they're aware that the feature is ready for QA? Also, worth adding that if it's gnarly to QA (e.g., lots of screenshots) we try to QA in a Slack thread

Reply:
Ooooo, I like that -- lemme add something and get another looksee from you

Reply:
* **Delivered**: The pull request has been merged and deployed to a QA environment. The developer has verified their work on this environment and added QA / testing notes to the ticket. At this stage, there will be review by Product, Design, and/or Data. The developer should @ mention any reviewers to make sure they're aware that the feature is ready for QA. Just as we expect developers to keep their ticket statuses up to date, we expect the same from Reviewers (Unstarted, In Review, Pass, or Revise). If the QA requires cross-team coordination or is particularly involved (e.g., lots of screenshots, states and verifications of outcomes), it is best to QA in a Slack thread (and link the thread in the ticket).